flex_model/core/wrapper.py

```diff
@@ -32,6 +40,48 @@
 logger = logging.getLogger(__name__)
 
 
+_OFFLOAD_MODES: Set[str] = {
+    "CPU",
+    "GPU",
+}
```

Comment:
Move into `FlexModel` as an attribute, or get rid of it and try to reuse the modes defined in `HookFunction`.